README.md

```diff
@@ -139,6 +139,15 @@ If you plan to release your application you'll need to publish it using the same
 
 ## Example applications
 
+As Decidim is a ruby gem, you can see the [https://github.com/decidim/decidim/network/dependents?type=application](dependent repostiories) to see how many applications are on the wild or tests that others developers have made. Here's a partial list with some of the projects that have used Decidim: 
+
```

Comment:
* s/As Decidim/Since Decidim/ -> It reads better IMO. 
* s/you can see/you can check out/ -> To avoid repetition of the same verb twice in the sentence. 
* s/repostiories/repositories/